src/DynamoCoreWpf/UI/Converters.cs

```diff
@@ -1769,6 +1769,19 @@ public object ConvertBack(object value, Type targetType, object parameter, Cultu
         }
     }
 
+    public sealed class NullToPinWidthConverter : IValueConverter
+    {
+        public object Convert(object value, Type targetType, object parameter, CultureInfo culture)
+        {
+            return value == null ? 0 : 4;
```

Comment:
Can you define `4` as const pin width here? Thanks!

Reply:
Addressed in 5c96555